src/modules/services_manager_service.rb

```diff
@@ -86,38 +90,40 @@ def load_units
           service.chomp! SERVICE_SUFFIX
           units[service] = {
             :status => status,
-            # bsc#956043 service can be 'just being activated' or 'reloaded'
-            # See https://github.com/systemd/systemd/blob/7152869f0a4a4612022244064cc2b3905b1e3fc7/src/basic/unit-name.c#L844
-            :active => (
-              active == Status::ACTIVE ||
-              active == Status::ACTIVATING ||
-              active == Status::RELOADING
-            ),
             :description => description.join(' ')
           }
         end
       end
 
-      def extract_services_from_unit_files
-        supported_unit_files.each do |name, status|
-          next if services[name]
+      def extract_services
+        @supported_unit_files.each do |name, status|
           services[name] = DEFAULT_SERVICE_SETTINGS.clone
-          services[name][:enabled] = status == Status::ENABLED
-          services[name][:active] = Yast::Service.Status(name).zero?
+          if @supported_units[name]
+            # Services are loaded into the system. Taking that one because there are more
+            # information
+            services[name][:loaded] = @supported_units[name][:status] == Status::LOADED
+            services[name][:description] = @supported_units[name][:description]
+          end
+          services[name][:can_be_enabled] = status == Status::STATIC ? false : true
         end
-      end
 
-      def extract_services_from_units
-        supported_units.each do |name, attributes|
+        # Add old LSB services (Services which are loaded but not available as a unit file)
+        @supported_units.each do |name, status|
+          next if services[name]
           services[name] = DEFAULT_SERVICE_SETTINGS.clone
-          if supported_unit_files[name]
-            services[name][:enabled] =  supported_unit_files[name] == Status::ENABLED
-          else
-            services[name][:enabled] = Yast::Service.Enabled(name)
+          services[name][:loaded] = @supported_units[name][:status] == Status::LOADED
+          services[name][:description] = @supported_units[name][:description]
+        end
+
+        # Rest of settings
+        services.each_key do |name|
+          services[name][:enabled] = Yast::Service.Enabled(name)
+          services[name][:active] = is_active?(name)
+          if !services[name][:description] || services[name][:description].length == 0
```

Comment:
NP: What about `services[name][:description].empty?`? Feel free to ignore this comment anyway :)